client/js/constants.js

```diff
@@ -1,5 +1,24 @@
 "use strict";
 
+const colorCodeMap = [
+	[0, "White"],
```

Comment:
I just realized you don't need numbers here. This could simply be an array of color names.